pulp_container/tests/functional/api/test_push_signatures.py

```diff
@@ -75,7 +75,7 @@ def test_assert_signed_image(
         assert decrypted.key_id == keyid
         assert decrypted.fingerprint == fingerprint
         assert decrypted.status == "signature valid"
-        assert decrypted.username == "Pulp QE"
+        assert decrypted.username == "pulp-fixture-signing-key"
```

Comment:
Oh, this will introduce a bad version interdependence issue. If we cannot defer the userid of the key, we should not assert on it.

Reply:
This test is useful for asserting data stored in Pulp. We verify whether we did not change anything along the way while parsing images. Am I reading it correctly that you want to remove this check?

Reply:
Depending on the version of pulpcore used, this username will be the new one or the old one. So the best way would be to find a way to get the asserted name from the gpgkey fixture in pulpcore. If that is impossible, i'd really just remove the assertion (or assert only that it's not empty).

Reply:
The fixture doesn't provide that information.  It does provide the KeyID, which IMO is good enough and effectively tests the same thing.  Actually the "fingerprint" is just a subset of the "key ID" too, yeah?

Reply:
Sure, let's remove it!